### PR TITLE
SAMBRO AlertHub and some SAFIRE data

### PR DESCRIPTION
**Pull request by waidyanatha**
**SAMBRO**
- The main change is the creation of a cascade template SAMBRO/AlertHub. It includes all the relevant prepop data. 
- A small change was made to the cascade template SAMBRO/Demo by moving the cap_alerting_authorities.csv file inside the Demo folder rather than sitting in the default SAMBRO folder, because this data is strictly for demo purpose. 
- the new template and propop data was tested and the working instance is now live here: http://cap.sahana.io:8000/eden/default/index

**SAFIRE**
- Also includes prepop demo data for SAFIRE.

_apologize for mixing up SAFIRE work in with this SAMBRO pull request. I didn't know how to exclude it._

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -28,3 +28,9 @@
 /modules/plugins/*
 !/modules/plugins/__init__.py
 !/modules/plugins/skeleton.py
+#
+#temporary file
+modules/templates/SAFIRE/Demo/scenario.csv
+modules/templates/SAFIRE/Demo/scenario_asset.csv
+modules/templates/SAFIRE/Demo/scenario_human_resource.csv
+modules/templates/SAFIRE/Demo/scenario_task.csv
```
Comment (flavour) [inline .gitignore]: I don't think we want these lines in trunk?

**Comment by flavour** [comment]
As well as removing the .gitignore lines, could you rebase & compress these 12 commits into 1? (This makes the changelog cleaner in trunk)

**Comment by waidyanatha** [comment]
I'm not very familiar with rebase; I tried this: https://git-scm.com/book/en/v2/Git-Branching-Rebasing but it only allows me to move the Head back once. I presume the easiest would be to delete my fork: waidyanatha/eden and start a new one from sahana/eden. Then clone the new fork waidyanatha/eden in my local machine and replace the SAMBRO folder with all the changes that were made. Would that solve this? Open to other suggestions?

**Comment by flavour** [comment]
Instructions for squashing commits are here:
http://eden.sahanafoundation.org/wiki/DeveloperGuidelines/Git#Squashingcommits

But yes, it might be easier, this time at least, to do the fresh clone approach...up to you :)

**Comment by waidyanatha** [comment]
An alternate PR was submitted and was merged: https://github.com/sahana/eden/pull/1437

**Pull request closed without merging**